Xamarin/DrawXShared/ViewControllerShared.cs

```diff
@@ -36,7 +37,7 @@ public class ViewControllerShared : UIViewController
 
         public ViewControllerShared(IntPtr handle) : base(handle)
```

Comment:
I may be missing something, but why is this in Shared? How does this compile on Android?